app/assets/javascripts/angular/controllers/orders_controller.js

```diff
@@ -0,0 +1,37 @@
+(function() {
+var app = angular.module('taxi', ['ngRoute']);
+
+app.controller('CreateOrderController', ['$scope', '$http', function($scope, $http) {
+    $scope.order = {};
+        $scope.phone_pattern = /(0)[0-9]{9}/;
+        $scope.email_pattern = /^(([-a-z0-9._]+)@([-a-z]+))(\.[a-z]+)?(\.[a-z]+)?$/;
+
+    $scope.addOrder = function() {
+    $scope.order = {
```

Comment:
Why this code is creating new property `order` in `$scope.order` with values that are already existsing in the `$scope.order`?